plone/app/querystring/querybuilder.py

```diff
@@ -189,33 +206,28 @@ def number_of_results(self, query):
         """Get the number of results"""
         results = self(query, sort_on=None, sort_order=None, limit=1)
         return translate(
-            _(u"batch_x_items_matching_your_criteria",
-              default=u"${number} items matching your search terms.",
-              mapping={'number': results.actual_result_count}),
-            context=self.request
+            _(
+                u"batch_x_items_matching_your_criteria",
+                default=u"${number} items matching your search terms.",
+                mapping={"number": results.actual_result_count},
+            ),
+            context=self.request,
         )
 
     def filter_query(self, query):
-        text = query.get('SearchableText', None)
+        text = query.get("SearchableText", None)
         if isinstance(text, dict):
-            text = text.get('query', '')
+            text = text.get("query", "")
         if text:
-            query['SearchableText'] = self.munge_search_term(text)
```

Comment:
I'd do the same way as for `1.x` keep the class method for better customization of the searchterm handling.